proto/lib/livebook_proto/secret_created.pb.ex

```diff
@@ -3,4 +3,5 @@ defmodule LivebookProto.SecretCreated do
 
   field :name, 1, type: :string
   field :value, 2, type: :string
+  field :deployment_group_id, 3, type: :string
```

Comment:
Shouldn't it be an integer and be optional?

Reply:
I see. As long as we declare them as strings in the schema, then it is fine by me. We don't need to care if they are integers here, we can treat the IDs as opaque (and strings are better for that indeed).

Reply:
It is! That should be a very small change on send_message on Teams.

Reply:
To clarify, I mean endpoints like org creation, which returns JSON with `org_id` and `user_id`. I think the main change would be a migration on Livebook side to convert ids stored in the hub struct in storage.

Reply:
Gah, Too much work. Given it is already an integer, it is not worth to change it preemptively. If we need to change it one day, then we do it.